Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingList/APIKeyDashboardList.swift

```diff
@@ -142,7 +142,8 @@ struct APIKeyDashboardList: View {
         .sheet(item: self.$presentedPaywall) { paywall in
             #if PAYWALL_COMPONENTS
             if let componentData = paywall.offering.paywallComponentsData {
-                TemplateComponentsView(paywallComponentsData: componentData)
+                // TODO: Get locale
```

Comment:
I think this todo can be removed! I think I added this 😅

Getting locale will happen internally in this component